airflow/providers/cncf/kubernetes/operators/pod.py

```diff
@@ -670,10 +678,15 @@ def write_logs(self, pod: k8s.V1Pod):
 
     def post_complete_action(self, *, pod, remote_pod, **kwargs):
         """Actions that must be done after operator finishes logic of the deferrable_execution."""
-        self.cleanup(
-            pod=pod,
-            remote_pod=remote_pod,
-        )
+        try:
+            self.cleanup(
+                pod=pod,
+                remote_pod=remote_pod,
+            )
+        except Exception:
+            # If one task got marked as failed, it should not raise exception which might cause retry.
```

Comment:
```suggestion
            # If task got marked as failed, it should not raise exception (which might cause retry).
```

nit: lets keep the comments consistent